clients/src/main/java/org/apache/kafka/clients/admin/KafkaAdminClient.java

```diff
@@ -1037,7 +1037,7 @@ private long sendEligibleCalls(long now) {
                 }
                 ClientRequest clientRequest = client.newClientRequest(node.idString(), requestBuilder, now,
                         true, requestTimeoutMs, null);
-                log.trace("Sending {} to {}. correlationId={}", requestBuilder, node, clientRequest.correlationId());
+                log.debug("Sending {} to {}. correlationId={}", requestBuilder, node, clientRequest.correlationId());
```

Comment:
I found this debug line very crucial; and in other clients (producer, consumer) the same line indicating when we actual put the request to the send queue was set at debug level as well, so I've changed it to also debug.